index.html

```diff
@@ -8,6 +8,8 @@
 copydoc: https://docs.google.com/document/d/1WQ3u4Eg9Vqp_yFBdIYUyGXkvhMxlPBMSr7g7O3rLzpg/edit
 ---
 
+<script async defer src="https://buttons.github.io/buttons.js"></script>
```

Comment:
No need for `async` here.
```suggestion
<script defer src="https://buttons.github.io/buttons.js"></script>
```